src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx

```diff
@@ -384,6 +392,32 @@ export const UploadPhotos: React.FC<UploadPhotosProps> = ({
                 disabled={isSubmitting || !isValid}
                 loading={isSubmitting || values.photos.some(c => c.loading)}
                 type="submit"
+                onClick={event => {
+                  if (!isLoggedIn) {
+                    event.preventDefault()
+
+                    showAuthDialog({
+                      mode: "Login",
```

Comment:
Since we don't often log out users from their sessions, I wonder if it makes more sense here to present the `Sign Up` mode instead

Reply:
Yeah, that's a good idea. We can default to the sign up modal instead. Existing users can easily switch to login from that. 👍🏼

Reply:
Switched to `Signup` mode in last commit 👍🏼